src/include/gurt/common.h

```diff
@@ -58,8 +58,24 @@
 extern "C" {
 #endif
 
-#define likely(x)       __builtin_expect((x), 1)
-#define unlikely(x)     __builtin_expect((x), 0)
+#ifndef likely
+#define likely(x)		__builtin_expect((x), 1)
+#endif
+#ifndef unlikely
+#define unlikely(x)		__builtin_expect((x), 0)
+#endif
+
+#ifdef MPL_USE_MEMORY_TRACING
+/* This is used to properly compile a MPICH romio plugins with --enable-g=mem */
+#define _gurt_calloc(a,b)	MPL_calloc(a,b,MPL_MEM_OTHER)
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)